docs/source/running.rst

```diff
@@ -101,3 +101,28 @@ While still inside the virtual environment, run::
                  --deployment-dir <contract-deployment-dir> \
                  --source-chain source \
                  --target-chain target
+
+Updating an agent
+-----------------
+
+To describe how to update an agent, it is worth to have a look on Beamer's versioning scheme. As described in
+branching strategy, each major version describes a different mainnet deployment thus a different set of contracts.
+Please note, that if you update your agent to a new major version, it will run on different contracts. Updating to a
+#new major version requires different steps for you to safely transition to a new major contract version.
+Any minor version upgrade indicates a new agent release (i.e. 1.X.0 -> 1.Y.0).
+
+Update to a new agent release
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Running your agent in a container, it is as easy as updating your image in the docker-compose.yml to the latest version.
+The repo https://github.com/beamer-bridge/run-your-own-agent is actively maintained and will provide you with the
+most uptodate agent version. Alternatively you will find the latest version under
+https://github.com/beamer-bridge/beamer/tags.
```

Comment:
Nitpick: I would instead link to https://github.com/beamer-bridge/beamer/releases which gives the release notes right away.